### PR TITLE
webapp/test/*.jsp: change to 2-space indentation #7352

### LINKED ISSUE #7352: *.jsp: change to 2-space indentation
Refactor all *.jsp pages to use 2-space indentation instead of 4.

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes (partially) #7352
Applied to all files in `src/main/webapp/test/*`

### CODE REVIEW

File: src/main/webapp/test/tableSort.jsp [hunk 1]
```diff
@@ -3,176 +3,180 @@
...
-                        <td >E +5%</td>
-                        <td >-1.667</td>
-                        <td >N/A</td>
-                    </tr>
-                </tbody>
-            </table>
-            <br> <br> <br>
-        </div>
+  <div class = "container">
+      <table class="table table-striped">
```
Comment (whipermr5) [inline src/main/webapp/test/tableSort.jsp]: Too much indentation here.

File: src/main/webapp/test/tableSort.jsp [hunk 2]
```diff
@@ -3,176 +3,180 @@
...
+                class="button-sort-none toggle-sort">Point <span class="icon-sort unsorted"></span></th>
+            <th id="button_sortPointNumber" data-toggle-sort-comparator="sortByPoints"
+                class="button-sort-none toggle-sort">Point (Number) <span class="icon-sort unsorted"></span></th>
+            <th id="button_sortDiff" data-toggle-sort-comparator="sortByDiff"
+                class="button-sort-none toggle-sort">Diff <span class="icon-sort unsorted"></span></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td >0</td>
```
Comment (whipermr5) [inline src/main/webapp/test/tableSort.jsp]: Might as well take this opportunity to fix the extraneous space after `<td` before `>`.

File: src/main/webapp/test/tableSort.jsp [hunk 3]
```diff
@@ -3,176 +3,180 @@
...
+            <td >Le Minh Khue</td>
+            <td >06 April 2011</td>
+            <td >E +5%</td>
+            <td >-1.667</td>
+            <td >N/A</td>
+          </tr>
+        </tbody>
+      </table>
+      <br> <br> <br>
     </div>
```
Comment (whipermr5) [inline src/main/webapp/test/tableSort.jsp]: Can take this opportunity to remove this extraneous `</div>` too.

Comment (sujeet14108) [inline src/main/webapp/test/tableSort.jsp]: Done :)

File: src/main/webapp/test/mashup.jsp [hunk 4]
```diff
@@ -8,429 +8,519 @@
...
+    </style>
+    <script type="text/javascript" src="<%= FrontEndLibrary.JQUERY %>"></script>
+    <script type="text/javascript" src="<%= FrontEndLibrary.BOOTSTRAP %>"></script>
+    <script type="text/javascript" src="/js/common.js"></script>
+  </head>
+  <body>
+    <div class="container theme-showcase">
+      <div class="row">
+        <div id="dhtmltooltip">
+        <div class="alert alert-warning">
```
Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: Needs indentation.

File: src/main/webapp/test/mashup.jsp [hunk 5]
```diff
@@ -8,429 +8,519 @@
...
-    </script>
+      </div>
+
+      <div class="intro">
+        <h1>Introduction</h1>
+        <table class="table table-striped">
+          <tr>
+            <td>
+              <p>This page lists all of the pages which make up the TEAMMATES app. It is used for manually
+                 inspecting the TEAMMATES UI for visual defects. To load this page, you will need to run the test
```
Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]:
```
<p>
  This...
  inspecting...
  ...
</p>
```

File: src/main/webapp/test/mashup.jsp [hunk 6]
```diff
@@ -8,429 +8,519 @@
...
+                 <strong>MashupPageUiTest.java</strong>.</p>
+            </td>
+          </tr>
+        </table>
+       </div>
+        <h1>Table Of Contents</h1>
```
Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: Too much indentation.

File: src/main/webapp/test/mashup.jsp [hunk 7]
```diff
@@ -8,429 +8,519 @@
...
+                  <a href="#studentFeedbackSubmitPage">Student Feedback Submit Page</a>
+                </li>
+                <li>
+                  <a href="#studentFeedbackQuestionSubmitPage">Student Feedback Question Submit Page</a>
+                </li>
+                <li>
+                  <a href="#studentFeedbackResultsPage">Student Feedback Results Page</a>
+                </li>
+              </ul>
+            <td>
```
Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: Missing a `</td>` before this?

File: src/main/webapp/test/mashup.jsp [hunk 8]
```diff
@@ -8,429 +8,519 @@
...
+              <p>This page lists all of the pages which make up the TEAMMATES app. It is used for manually
+                 inspecting the TEAMMATES UI for visual defects. To load this page, you will need to run the test
+                 <strong>MashupPageUiTest.java</strong>.</p>
+            </td>
+          </tr>
+        </table>
+       </div>
+        <h1>Table Of Contents</h1>
+        <table class="table table-striped">
+        <tbody>
```
Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: Needs more indentation.

File: src/main/webapp/test/mashup.jsp (hunk 4 above)
Comment (sujeet14108) [inline src/main/webapp/test/mashup.jsp]: @whipermr5 One of the `</div>` is missing in original file 94 `<div` and 93 `</div`
Which div is unclosed ` line 42 `?

Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: Looks like `<div id="dhtmltooltip">` is unclosed. A [search](https://github.com/TEAMMATES/teammates/search?utf8=%E2%9C%93&q=dhtmltooltip&type=) in the repo shows that `commonAssertRegexTestPage.html` uses it like this: `<div id="dhtmltooltip"></div>`.

File: src/main/webapp/test/mashup.jsp (hunk 6 above)
Comment (sujeet14108) [inline src/main/webapp/test/mashup.jsp]: Isn't it on correct indentation level ?

Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: Well, it's not aligned with `</div>`, so either one is wrong.

**Comment by sujeet14108** [comment]
Done

File: src/main/webapp/test/mashup.jsp [hunk 9]
```diff
@@ -8,429 +8,520 @@
...
+                  <a href="#errorPage">Error Page</a>
+                </li>
+                <li>
+                  <a href="#entityNotFoundPage">Entity Not Found Page</a>
+                </li>
+              </ul>
+            </td>
+          </tr>
+          </tbody>
+        </table>
```
Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: This and the above line have too much indentation.

File: src/main/webapp/test/mashup.jsp [hunk 10]
```diff
@@ -8,429 +8,520 @@
...
+
+        <div class="pageinfo">Instructor Home Page</div>
```
Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: After the above is fixed, this should be two spaces to the left.

File: src/main/webapp/test/mashup.jsp [hunk 11]
```diff
@@ -8,429 +8,520 @@
...
+        <div id="errorPage"></div>
+        <br><hr class="hr-bold"><br>
+
+        <div class="pageinfo">Entity Not Found Page</div>
+        <div id="entityNotFoundPage"></div>
+        <br><hr class="hr-bold"><br>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
```
Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: After the above is fixed, this should be two spaces to the left.

File: src/main/webapp/test/mashup.jsp [hunk 12]
```diff
@@ -8,429 +8,520 @@
...
+      </div>
```
Comment (whipermr5) [inline src/main/webapp/test/mashup.jsp]: Extraneous `</div>`. There are actually 93, not 94, opening `<div>`s - there's a false positive in the `<script>` section.

File: src/main/webapp/test/tableSort.jsp [hunk 13]
```diff
@@ -3,176 +3,179 @@
...
+        </tr>
+
+        <tr>
+          <td>2</td>
+          <td>Hou GuoChen</td>
+          <td>02 January 2012</td>
+          <td>E -21%</td>
+          <td>0.333</td>
+          <td>+3%</td>
+          </tr>
```
Comment (whipermr5) [inline src/main/webapp/test/tableSort.jsp]: Too much indentation.

**Comment by sujeet14108** [comment]
@whipermr5 Next folder to do is `/src/main/webapp/partials/*` inside this that there is file `instructorHelpSessions.jsp` which is 8000 lines. Should i do that file with all other files or separately ?

**Comment by whipermr5** [comment]
@sujeet14108 Perhaps you can break up that file into multiple files first (one for each question type).

File: src/main/webapp/test/allJsUnitTests.jsp [hunk 14]
```diff
@@ -3,36 +3,36 @@
 <!DOCTYPE html>
 <html>
 <head>
```
Comment (whipermr5) [inline src/main/webapp/test/allJsUnitTests.jsp]: Just noticed this.

File: src/main/webapp/test/allJsUnitTests.jsp [hunk 15]
```diff
@@ -3,36 +3,36 @@
...
+  <script type="text/javascript" src="<%= FrontEndLibrary.JQUERY_UI %>"></script>
+  <script type="text/javascript" src="<%= FrontEndLibrary.BOOTSTRAP %>"></script>
+  <script type="text/javascript" src="<%= FrontEndLibrary.BOOTBOX %>"></script>
+  <script type="text/javascript" src="<%= FrontEndLibrary.TINYMCE %>"></script>
+  <script type="text/javascript" src="<%= FrontEndLibrary.QUNIT %>"></script>
+  <script type="text/javascript" src="<%= FrontEndLibrary.BLANKET %>"></script>
 
-    <script type="text/javascript" src="/test/jsUnitTests.js" data-cover></script>
+  <script type="text/javascript" src="/test/jsUnitTests.js" data-cover></script>
 </body>
```
Comment (whipermr5) [inline src/main/webapp/test/allJsUnitTests.jsp]: Marking this.

File: src/main/webapp/test/tableSort.jsp [hunk 16]
```diff
@@ -3,176 +3,179 @@
 <!DOCTYPE html>
 <html>
 <head>
```
Comment (whipermr5) [inline src/main/webapp/test/tableSort.jsp]: Marking this.

File: src/main/webapp/test/tableSort.jsp [hunk 17]
```diff
@@ -3,176 +3,179 @@
...
+          <td >06 April 2011</td>
+          <td >E +5%</td>
+          <td >-1.667</td>
+          <td >N/A</td>
+        </tr>
+      </tbody>
+    </table>
+    <br> <br> <br>
+  </div>
 </body>
```
Comment (whipermr5) [inline src/main/webapp/test/tableSort.jsp]: Marking this.

File: src/main/webapp/test/tableSort.jsp [hunk 18]
```diff
@@ -3,176 +3,179 @@
...
-            </table>
-            <br> <br> <br>
-        </div>
-    </div>
+  <div class = "container">
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th id="button_sortid"
+              class="button-sort-ascending toggle-sort">ID <span class="icon-sort unsorted"></span></th>
```
Comment (whipermr5) [inline src/main/webapp/test/tableSort.jsp]: Can we do:
```
<th id="button_sortid" class="button-sort-ascending toggle-sort">
  ID <span class="icon-sort unsorted"></span>
</th>
```
for this and similarly for the rest?

**Comment by sujeet14108** [comment]
@whipermr5 Have a look

**Comment by whipermr5** [comment]
@sujeet14108 Have you done a visual comparison? There should be no visible changes before or after; not a single pixel.

**Comment by sujeet14108** [comment]
@whipermr5 I have seen only `http://localhost:8888/test/mashup.jsp` manually. Is there any tool to check change ?

**Comment by whipermr5** [comment]
@sujeet14108 I don't know of any (you should try searching if you really want one). Personally I'd do it manually though.

**Pull request merged**